manifests/image.pp

```diff
@@ -74,10 +75,17 @@
     $image_find    = "${docker_command} images | cut -d ' ' -f 1 | egrep '^(docker\\.io/)?${image}$'"
   }
 
+  if $build_args {
+    validate_array($build_args)
+    $build_args_string = join ( regsubst ($build_args, '^(.*)$', '--build-arg \1 '), '')
+  } else {
+    $build_args_string = ''
+  }
+
   if $docker_dir {
-    $image_install = "${docker_command} build -t ${image_arg} ${docker_dir}"
+    $image_install = "${docker_command} build ${build_args_string}-t ${image_arg} ${docker_dir}"
```

Comment:
Ah, no worries. I'm just fixing the rake issue, rake 11.x shipped this weekend. You should be able to rebase with master once I get that fixed.